packages/manager/cypress/e2e/core/domains/smoke-create-domain-records.spec.ts

```diff
@@ -1,40 +1,33 @@
-/* eslint-disable sonarjs/no-duplicate-string */
 import { authenticate } from 'support/api/authentication';
 import { createDomain } from 'support/api/domains';
-import { fbtClick, getClick } from 'support/helpers';
 import { interceptCreateDomainRecord } from 'support/intercepts/domains';
-import { cleanUp } from 'support/util/cleanup';
 import { createDomainRecords } from 'support/constants/domains';
 
 authenticate();
-describe('Creates Domains record with Form', () => {
-  before(() => {
-    cleanUp('domains');
-  });
 
-  createDomainRecords().forEach((rec) => {
-    return it(rec.name, () => {
-      createDomain().then((domain) => {
-        // intercept create api record request
-        interceptCreateDomainRecord().as('apiCreateRecord');
-        const url = `/domains/${domain.id}`;
-        cy.visitWithLogin(url);
-        cy.url().should('contain', url);
-        fbtClick(rec.name);
+describe('Creates Domains records with Form', () => {
+  it('Adds domain records to a newly created Domain', () => {
+    createDomain().then((domain) => {
+      // intercept create api record request
+      interceptCreateDomainRecord().as('apiCreateRecord');
+      const url = `/domains/${domain.id}`;
+      cy.visitWithLogin(url);
+      cy.url().should('contain', url);
+    });
+
+    createDomainRecords().forEach((rec) => {
+      cy.findByText(rec.name).click();
+      rec.fields.forEach((f) => {
+        cy.get(f.name).type(f.value);
+      });
```

Comment:
```suggestion
      rec.fields.forEach((field) => {
        cy.get(field.name).type(field.value);
      });
```

I know this is existing code. I'm not a fan of using the alphabet as variable names, so a minor suggestion for clarity here.

Reply:
I agree with you on this. I've updated the names.